### ISSUE TITLE
Change triple-state checkbox into radio buttons.

### ISSUE THREAD

**Issue by mxxcon** [facebook/Rapid#1299]
### Description

On some of the templates you use a triple-state checkbox, for example "Tactile Paving" option on curbs: Yes, No, Neutral.
You need to click 1-3 times to cycle through the states to get the desired result.

I think it would be better to change such a template into something along the lines of radio buttons the way you have it on "Wheelchair Access".

That way you can click only once directly on the desired state without having to cycle through all of them.

While saving 1 or 2 clicks might not sound much, when you are mapping 8 curbs per intersection, such benefits add up.

This can apply not just to curbs, but other templates that similarly use triple-state checkboxes.

### Screenshots

![image](https://github.com/facebook/Rapid/assets/847140/c085cd40-969b-4e6f-b479-6ae527003313)

### Version

2.2.4

### What browser are you seeing the problem on? What version are you running?

Chrome v121.0

### The OS you're using

win

### Steps to reproduce

Open kerbs template
Click on tactile paving option.

### The browser URL at the time you encountered the bug

https://rapideditor.org/edit?#background=EsriWorldImagery&datasets=fbRoads,msBuildings&disable_features=buildings,boundaries&map=20.49/40.60165/-73.99744

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36

**Comment by bhousel** [comment]
Good suggestion, I can see how it's annoying to need to keep clicking this.

Did you see we added some presets in #1146 for lowered kerb + tactile paving ? 
It might be faster to just use the preset picker. 🤔

**Comment by bhousel** [comment]
Also wanted to mention, you can shift-click to select a bunch of kerb ramps and retag them all at once.  

I do agree with your point though, saving clicks is good, and we are open to changing what fields appear in the sidebar.

**Comment by mxxcon** [comment]
> Also wanted to mention, you can shift-click to select a bunch of kerb ramps and retag them all at once.

Yes, I'm aware of that functionality, but since RapiD can so easily lose selection if I accidentally click on an empty space, I don't tend to use it. It's too frustrating and often I end up spending more time and more clicks re-selecting things.

**Comment by yaomtc** [comment]
This would also solve my other problem: when I click the tactile paving checkbox, it moves to the bottom of the fields section. In order to set it to "no", I have to click it once, move the cursor down to its new position, and check it again. Should I file a separate bug for that?
